### PR TITLE
Adds raw json links to results

### PR DESCRIPTION
**Pull request by kognate**
Results tables now include raw json links
that open in a new tab.

### CODE REVIEW

**Comment by nfriedly** [comment]
eslint ;)

File: views/includes/underscore-templates.jade [hunk 1]
```diff
@@ -49,6 +49,10 @@ script(type='text/html', id='trainPreviewImagesTemplate').
 
 script(type='text/html', id='useResultsTableTemplate').
   <div class="results-table--container">
+    <div class="results--json">
+      <a target="_blank" class="json">JSON <img src="/images/link_image.png"></a>
```
Comment (nfriedly) [inline views/includes/underscore-templates.jade]: Is `link_image.png` supposed to be included in this PR?

File: app.js [hunk 2]
```diff
@@ -189,6 +189,11 @@ app.post('/api/classify', app.upload.single('images_file'), function(req, res) {
       if (req.body.classifier_id) {
         combine.value[0].classifier_ids = req.body.classifier_id;
       }
+
+      combine.value[0].raw = {};
+      methods.map(function(methodName, idx) {
+        combine.value[0].raw[methodName] = encodeURIComponent(JSON.stringify(results[idx].value[0]));
+      });
       res.json(combine.value[0]);
```
Comment (nfriedly) [inline app.js]: This works but JSON-encoded-JSON is sort of an anti-pattern. I'd recommend refactoring the entire thing a bit: Given that you now want the raw JSON on the client, you might as well just send that and let the client pick out what it needs.

Additionally, this endpoint seems overly complex, probably because it's trying to do two different tasks with a single endpoint - either classify with a custom classifier, or classify with the default one + detect text and faces. Splitting those into two separate endpoints would make things simpler.

It might also be nice to pretty-print the JSON, but, again, I'd recommend just sending the raw JSON down and doing the stringify and pretty-printing on the client.

Not a blocker, just something to consider.

Comment (kognate) [inline app.js]: pretty-printing the json broke the underscore templates in a way I didn't understand.  That's also the thing that complicates a refactoring of this method,  it's pretty tightly coupled to the underscore templates.  I'd rather refactor it all (templates and endpoint), but that's not something that can reasonably be done in this sprint.

File: views/includes/underscore-templates.jade (hunk 1 above)
Comment (kognate) [inline views/includes/underscore-templates.jade]: Yes,  I've added it.  I put it into another pull request but that was dumb.

File: app.js (hunk 2 above)
Comment (nfriedly) [inline app.js]: Fair enough.
